### DISCUSSION TITLE
chpaters.re ファイルにフォルダー パスを含めることはできますか?

### DISCUSSION THREAD

**Discussion by jasondee1992** [kmuto/review#1851] [Q&A]
chpaters.re ファイルにフォルダー パスを含めることはできますか?

すべての .re ファイルを 1 つのフォルダーに入れる予定です。

![image](https://user-images.githubusercontent.com/49851974/183323950-41ba3245-f9e9-4257-8a86-4552ed5b78f9.png)

**Comment by kmuto** [comment]
catalog.ymlでフォルダを指定することはできません(ビルドできることもありますが、おそらくrake pdfの場合は内容が含まれない結果になります)。
「すべてのreファイルを特定のフォルダに入れ、config.ymlのcontentdirでそのフォルダを指定する」ことはできます。
